test/formio-wrapper-events.test.js

```diff
@@ -77,6 +79,7 @@ describe('Formio Wrapper Event Tests.', () => {
     assert.called(spiedFireExtraEvent);
     assert.calledTwice(spiedGoToPreviousPage);
     assert.called(spiedGoToPage);
+    assert.called(spiedRepopulateIngridientDataGrid);
```

Comment:
unsure how this assert would show that it 'added' back the user ingredients.

Reply:
Removed this assert call